galaxy_importer/collection.py

```diff
@@ -220,6 +222,19 @@ def _build_contents_blob(self):
 
     def _build_docs_blob(self):
         """Build importer result docs_blob from collection documentation."""
+
+        # return an empty DocsBlob if run_ansible_doc=False
+        # if not self.cfg.run_ansible_doc:
```

Comment:
not sure if this comment line is meant to stick around

Reply:
+1, please don't leave commented unused blocks of code.